### PR TITLE
https://issues.redhat.com/browse/ACM-12053 IBM Z Subm

### PR DESCRIPTION
**Pull request by oafischer**
2.11 only, Train GA

### CODE REVIEW

**Comment by aswinsuryan** [comment]
Looks good. @Jaanki Do you want to hold this PR until the implementation is verified?

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -77,6 +77,8 @@ See link:../business_continuity/backup_restore/backup_intro.adoc#backup-intro[Ba
 [#net-whats-new]
 == Networking
 
+You can now deploy Submariner on IBM Z by using the console. See link:../networking/submariner/deploy_subm_console.adoc#deploying-submariner-console[Deploying Submariner by using the console] to learn more.
```
Comment (jc-berger) [inline release_notes/whats_new.adoc]: for usability, I find links easier to click if they're at the end of the sentence. I also think leading with the action prompts the user to click it more efficiently. 

```suggestion
You can now deploy Submariner on IBM Z by using the console. To learn more, see link:../networking/submariner/deploy_subm_console.adoc#deploying-submariner-console[Deploying Submariner by using the console].
```

**Review by jc-berger — APPROVED** [review]
/lgtm

**Comment by Jaanki** [comment]
> Looks good. @Jaanki Do you want to hold this PR until the implementation is verified?

Yes, please. Lets wait for the IMB Z team to verfiy this first before publishing.

**Comment by oafischer** [comment]
GA is soon, is there any update @Jaanki ?

**Comment by oafischer** [comment]
No QE confirmation, closing PR.

**Pull request closed without merging**